keyboards/skeletonkbd/frost68/keymaps/default/keymap.c

```diff
@@ -0,0 +1,26 @@
+// Copyright 2023 skeletonkbd (@skeletonkbd)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names { _BASE, _L1, _L2, _L3 };
```

Comment:
Only using 2 layers, only need two defined. 
```suggestion
enum layer_names { 
    _BASE, 
    _L1,
};
```